edgePy/io/_io.py

```diff
@@ -42,10 +80,14 @@ def read_file(self):
                 line = line.strip()
                 if not header_read:
                     self.samples = line.split("\t")
+                    self.clean_headers()
                     header_read = True
                 else:
                     self.raw_data.append(line.split("\t"))
 
+    def clean_headers(self):
+        self.samples = [s.replace("\"", "").strip() for s in self.samples]
```

Comment:
Consider making this a `@staticmethod` as it doesn't need to rely on `self`.

For instance, if a user runs `clean_headers()` once, the internal data structure will change. 

However, if a user runs the method a second time, the internal data structure is left unchanged. 

Having a method switch between altering internal state and then becoming idempotent is confusing.

```python
class DataImporter(object):
    def __init__(self, filename=None):
        self.read_file(filename)
        ...

    @staticmethod
    def clean_headers(headers):
        ...
        return headers

    def read_file(self, filename):
        headers, contents = ...
        self.headers = self.clean_headers(headers)
```